index.html

```diff
@@ -163,12 +239,28 @@
       .attr('r', 5);
 
     points
-      .attr('cx', function (d) { return x(d.unemployment); })
-      .attr('cy', function (d) { return y(d.inflation); });
+      .attr('cx', function (d) { return x(d.year); })
+      .attr('cy', function (d) { return y(d.value); });
 
     points.exit().remove();
 
     // HISTOGRAMS
+    var hist = d3.layout.histogram()
+       .value(function (d) { return d.unemployment; })
+       .bins(x.ticks(binCount)
```

Comment:
Note you still have d.unemployment here. It might help to try and recreate code in smaller increments so it's easier to keep track of changes like this one.